src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -176,6 +188,14 @@ public String getHmacSecret(final String countryIsoCode) {
         return secretMap.get(adjustCountryCode(countryIsoCode));
     }
 
+    public String getHmacAlgorithm(final String countryIsoCode) {
+        if (countryIsoCode == null || hmacAlgorithmMap.isEmpty()) {
```

Comment:
Do we have to be cautious about the `UK` vs `GB` confusion? I guess whatever Adyen sees is what you we should use inside this plugin.